src/Schema/Factories/FieldFactory.php

```diff
@@ -162,17 +163,10 @@ function () use ($argumentMap, $resolverWithMiddleware) {
                 $this->flushValidationErrorBuffer();
 
                 $argumentSet = $this->typedArgs->fromResolveInfo($this->args, $this->resolveInfo);
-                $modifiedArgumentSet = $argumentSet
-                    ->spread()
-                    ->rename();
-                $this->resolveInfo->argumentSet = $modifiedArgumentSet;
-
-                return $resolverWithMiddleware(
-                    $this->root,
-                    $modifiedArgumentSet->toArray(),
-                    $this->context,
-                    $this->resolveInfo
-                );
+                $spreadArguments = $argumentSet->spread();
```

Comment:
Why did you remove the rename operation?

Reply:
I think this went wrong when trying to rest all style changes. I'll fix it.